views/forms/BarChartFields.jsx

```diff
@@ -4,14 +4,21 @@ import TextField from '@material-ui/core/TextField'
 import Button from '@material-ui/core/Button'
 import Typography from '@material-ui/core/Typography'
 import ColorPicker from '../components/ColorPicker'
+import Checkbox from '@material-ui/core/Checkbox'
 
 import { colors } from '../constants/styles'
 
 import { targetValuesFields } from '../fe-utils/targetValuesUtil'
 
 const BarChartFields = ({ classes, formValues, setFormValues, studies }) => {
-  const updateFormValues = (e) =>
-    setFormValues({ ...formValues, [e.target.name]: e.target.value })
+  const updateFormValues = (e) => {
+    console.log(e.target.name, e.target.checked)
+    setFormValues({
+      ...formValues,
+      [e.target.name]:
+        e.target.name !== 'public' ? e.target.value : e.target.checked,
```

Comment:
Could check against `e.target.type === 'checkbox'` in case other check boxes are added or the name of this one changes
```suggestion
        e.target.type === 'checkbox' ? e.target.checked : e.target.value
```

Reply:
good catch.
[a9d4afb](https://github.com/AMP-SCZ/dpdash/pull/143/commits/a9d4afbc6bffe85e5131e8547ab31a657102d5a9)

Reply:
👍 Looks good
Something I've found helpful is to push a separate commit for each comment in code review (assuming it warrants a code change 😄 ), they can be squashed later prior to merging, but it makes it really easy for the reviewer to view just the relevant diff for a conversation.

Reply:
Thanks :)